File: javascript/pangram/pangram.js
function Pangram (sentence) {
  this.sentence = sentence;
}

var strip = function (sentence) {
  return sentence.toLowerCase().replace(/[^a-z]/g, '')
};
var alphabet = {
    'ascii' : 'abcdefghijklmnopqrstuvwxyz'
}
Pangram.prototype.isPangram = function () {
  let sentence = this.sentence;
  let sentenceToCheck = strip(sentence);
  var i;
  for (i = 0; i < alphabet['ascii'].length; i++){
    if (!sentenceToCheck.includes(alphabet['ascii'][i])){
      return false;
    }
  }
  return true;
};

module.exports = Pangram;

File: python/pangram/pangram.py
import re

def is_pangram(sentence):
    stripped = stripSentence(sentence)
    alphabet = 'abcdefghijklmnopqrstuvwxyz'

    for i in range(len(alphabet)):
        if not alphabet[i] in stripped:
            return False
    return True

def stripSentence(sentence):
    sentence = sentence.lower()
    result = re.findall(r"[a-zA-Z]", sentence)
    return result
